### PR TITLE
frontend-feat: add dynamic remote banner action & home screen ctrl split

### PR DESCRIPTION
**Pull request by spravinkumar9952**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added integration tests for my changes where possible
- [ ] I tested the changes using integration tests
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-common/src/RemoteConfig/Types.purs [hunk 1]
```diff
@@ -16,10 +16,10 @@ module Common.RemoteConfig.Types where
 
 import Prelude
 import Data.Maybe (Maybe)
-import Foreign.Class (class Decode, decode)
+import Foreign.Class
 import Data.Generic.Rep (class Generic)
 import Data.Newtype (class Newtype)
-import Presto.Core.Utils.Encoding (defaultDecode)
+import Presto.Core.Utils.Encoding
```
Comment (witcher-shailesh) [inline Frontend/ui-common/src/RemoteConfig/Types.purs:22]: try not to remove the imports and use aliases for reference, code readability will be good in that sense

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controllers/Types.purs
Comment (witcher-shailesh) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controllers/Types.purs:1]: multiple unused imports are there in this file 😢

**Review by witcher-shailesh — CHANGES_REQUESTED** [review]
Is it tested thoroughly once

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controllers/Types.purs
Comment (spravinkumar9952) [reply Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controllers/Types.purs:1]: copied from controller 😢 . fixed now.

**Pull request merged**